scripts/SANS/sans/gui_logic/presenter/run_tab_presenter.py

```diff
@@ -302,6 +309,11 @@ def set_view(self, view):
                 self._table_model.column_name_converter.index('options_column_model'),
                 self._table_model.get_options_hint_strategy())
 
+            self._view.gui_properties_handler = SANSGuiPropertiesHandler({"user_file": (self._view.set_out_default_user_file,
+                                                                                        str)},
+                                                                         line_edits={"user_file":
+                                                                                     self._view.user_file_line_edit})
+
```

Comment:
You could move these arguments to a new line to avoid the PEP8 line length warning